tasks/fullStack/app/models/review.rb

```diff
@@ -1,3 +1,8 @@
 class Review < ApplicationRecord
   belongs_to :product
+
+  validates :product_id, presence: true
+  validates :reviewer_name, presence: true, length: { maximum: 100 }
+  validates :body, presence: true, length: { maximum: 1000 }
+  validates :rating, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 1, less_than_or_equal_to: 5 }
```

Comment:
Good idea adding validations 👍 

I wonder how `only_integer: true` would behave with the `float` field in the DB?

https://github.com/alexander-meyer/recruitment/blob/05c6fb69c91454f968f692c852b84a1d78021f70/tasks/fullStack/db/schema.rb#L26-L29

Reply:
good point; from my understanding, I believe the validation here would be enforced at the application-level, before it is saved to the DB (hence why it saves the object without issue).

I can imagine a future scenario where we want to support e.g. decimal ratings, at which point this validation would no longer be accurate. A good reminder to double-check the schema when adding validations like this 😅